protobuf/drand/common.proto

```diff
@@ -38,6 +38,7 @@ message GroupPacket {
     uint64 transition_time = 5;
     bytes genesis_seed = 6;
     repeated bytes dist_key = 7;
+    uint32 catchup_period = 8;
```

Comment:
Small comment indicating this is in second (i know it's not written everywhere else but better to do it than not :D )